style.css

```diff
@@ -1,37 +1,33 @@
-body {
+html {
   background-color: rgb(19, 19, 143);
-  font-size: 20px;
-  font-family: sans-serif, Arial, Helvetica;
-  margin: 0;
-  padding: 0;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
[REQUIRED] Please use the font family which is provided in Figma template
![image](https://user-images.githubusercontent.com/74744787/158881626-5213565a-0e39-48a7-8d0f-a78b1c1347b0.png)